cosmos-sdk-proto/src/prost/cosmos.auth.v1beta1.rs

```diff
@@ -88,17 +88,6 @@ pub mod query_client {
     pub struct QueryClient<T> {
         inner: tonic::client::Grpc<T>,
     }
-    impl QueryClient<tonic::transport::Channel> {
-        #[doc = r" Attempt to create a new client by connecting to a given endpoint."]
-        pub async fn connect<D>(dst: D) -> Result<Self, tonic::transport::Error>
-        where
-            D: std::convert::TryInto<tonic::transport::Endpoint>,
-            D::Error: Into<StdError>,
-        {
-            let conn = tonic::transport::Endpoint::new(dst)?.connect().await?;
-            Ok(Self::new(conn))
-        }
-    }
```

Comment:
This seems somewhat problematic...

Reply:
Anyway I can merge this? - I can close it if this is not going to happen

Reply:
I'm happy to merge this if it doesn't cause a regression for non-WASM use cases, but this appears to be a blocker